src/renderer/services/price-api/coin-cap.js

```diff
@@ -210,4 +196,33 @@ export default class CoinCapAdapter {
       max: max(Object.values(datasets))
     }
   }
+
+  /**
+   * Convert an amount of money to another currency
+   * @param {Number} amount
+   * @param {Object} options
+   * @return {Number} Returns conversion result
+   */
+  static __convert (amount, { from, to, base, rates }) {
+    if (from && to) {
+      // If `from` equals `base`, return the basic exchange rate for the `to` currency
+      if (from === base && has(rates, to)) {
```

Comment:
Does `has` only check if the key exists? If so it could be swapped out for `Object.prototype.hasOwnProperty.call(rates, to)`. It might be longer but saves on relying on external packages. In the future, we'd like to remove lodash completely (if possible)

Reply:
Good tip :+1: